src/main/webapp/Portal.html

```diff
@@ -159,382 +159,70 @@
       loadCss('//fonts.googleapis.com/css?family=Lato')
       loadCss('https://fonts.googleapis.com/icon?family=Material+Icons')
 
-      /*! head.load.min.js - v1.0.3 */
```

Comment:
I removed headjs. I was always curious what `head.load` did in this script, and I learned that it was useful in IE6, IE7, and other browsers of that era where scripts were not necessarily loaded in parallel. This is not something we need to worry about now, so I removed the script.